uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerMicrosoftAuthenticator.java

```diff
@@ -342,6 +342,11 @@ protected void performDeviceRegistrationHelper(@NonNull final String username,
         }
     }
 
+    public void shouldUseDeviceSettingsPage(final boolean shouldUseDeviceSettingsPage) {
```

Comment:
The name of this method sounds like it is supposed to return a boolean, however, I see that the method is `void`

Reply:
Okay, it seems this method is actually a setter. Should the method name be `setShouldUseDeviceSettingsPage` ?

Reply:
Ahh yes, it's not properly named. I'll fix that. Although I think I'll probably be getting rid of it fairly soon.